impl/client/src/main/javascript/web/prompting/PromptPanel.js

```diff
@@ -1161,6 +1162,7 @@ function(Base, Logger, DojoNumber, i18n, Utils, GUIDHelper, WidgetBuilder, Dashb
               this.dashboard.addComponent(errorComponent);
               this.dashboard.updateComponent(errorComponent);
               panel.components.splice(errIndex, 0, errorComponent);
+              panel.addErrorLabel(errorComponent, error);
```

Comment:
As discussed with @dcleao, added this function to add error message in a div label to the panel, avoiding a full reload of prompt panel